Jenkinsfile

```
@@ -3,10 +3,5 @@
 buildPlugin(useContainerAgent: true,
         configurations: [
                 [platform: 'linux', jdk: '11'],
-                [platform: 'windows', jdk: '8'],
-
-                // testing the Guava & Guice bumps
-                // https://github.com/jenkinsci/jenkins/pull/5707
-                // https://github.com/jenkinsci/jenkins/pull/5858
-                [ platform: "linux", jdk: "8", jenkins: '2.321' ]
+                [ platform: "linux", jdk: "17", jenkins: '2.387.2' ]
```

This should be testing Windows rather than Linux so that the tests cover both Windows and Linux and cover both Java 11 and Java 17.  

As a matter of style, we prefer single quotes instead of double quotes in Pipeline strings when the Pipeline string does not include a variable to be interpreted or a method call to be made when the string is computed.  Single quotes in groovy mean that the string is not to be interpolated by the groovy parser.  In this case, the result is exactly the same final string, but single quotes are consistent with the first line and with the example in the [tutorial](https://www.jenkins.io/doc/developer/tutorial-improve/add-a-jenkinsfile/#create-the-jenkinsfile).
```suggestion
                [ platform: 'windows', jdk: '17', jenkins: '2.387.2' ]
```